milestone-3/backend/index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const app = express()
+app.use(express.json())
+const mongoose = require('mongoose')
+const connection_url = "mongodb+srv://newUser:<newPassword>@websitecluster.syy0q.mongodb.net/RecipesDB?retryWrites=true&w=majority"
+
+mongoose.connect(connection_url)
+.then(() => console.log('Successfully connected'))
+.catch((error) => console.error(`Could not connect due to ${error}`))
+
+const Recipe = new mongoose.Schema({
+    recipe_name: String,
+    recipe_description: String
+}, {collection: 'Recipes'})
+
+const Setup = mongoose.model('Recipes', Recipe)
+module.exports = Setup
+app.use(express.json())
+
+app.get('/', (req, res) => {
+  res.send(`Hello world!`)
+})
+
+app.get("/api/recipe", async (req, res) => {
+    const recipes = await Recipe.find({})
+    res.send(recipes)
+})
+
+app.get("/api/recipe/:recipeName", async (req, res) => {
+    const recipes = await Recipe.find({"/:recipeName"})

```

Comment:
So when writing a mongoose query, the syntax can be a bit confusing. when you use the find() method, you want to find documents in your database that match whatever specifications you are looking for. The format for this is as follows:
find({ _fieldName_: **fieldValue** }). _fieldName_ would be whatever database document field you are trying to match. So in this case, you would probably want to use recipe_name since you are looking for a specific recipe based off its name.
**fieldValue** would be whatever specific value you are trying to find. So if you are looking for a taco recipe, this may be called 'tacos' or whatever it is in the database. This is the actual recipe name you are looking for.
Below is a link to a more detailed explanation of this. If this isn't clear enough and these recourses don't help feel free to reach out to me so I can help out! :)
https://www.notion.so/Writing-a-Query-b48e00b154544487af8df8a36acb1161